common/src/main/java/com/microsoft/hydralab/common/network/DummyNetworkMonitor.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.hydralab.common.network;
+
+import com.microsoft.hydralab.common.entity.common.DeviceInfo;
+import com.microsoft.hydralab.common.management.device.DeviceDriver;
+import com.microsoft.hydralab.common.util.ADBOperateUtil;
+import org.slf4j.Logger;
+
+import java.io.File;
+
+public class DummyNetworkMonitor implements NetworkMonitor {
+
+    public DummyNetworkMonitor() {
+    }
+
+    @Override
+    public void start() {
+    }
```

Comment:
Add some logs to declare this class is a dummy monitor.

Reply:
fixed